Formula/w/whisperkit.rb

```diff
@@ -0,0 +1,18 @@
+class Whisperkit < Formula
+  desc "Swift native on-device speech recognition with Whisper for Apple Silicon"
+  homepage "https://github.com/argmaxinc/WhisperKit"
+  url "https://github.com/argmaxinc/WhisperKit.git", tag: "v0.3.3", revision: "bfa357e897dadae0dca6314424bb119149e375d2"
+  license "MIT"
+
+  depends_on xcode: ["15.0", :build]
+  depends_on arch: :arm64
```

Comment:
What does it require ARM for?

Reply:
we want to restrict it just to M1 arch, it uses a CoreML model which is going to be slower on x86

Reply:
If it works on more platforms Homebrew should ship it for those too

Reply:
removed